intro-HPC/ch_troubleshooting.tex

```diff
@@ -20,14 +20,77 @@ \section{Walltime issues}
 \section{Out of quota issues}
 
 Sometimes a job hangs at some point or it stops writing in the disk. These errors are usually
-related to the quota usage. You may have reached your quota limit at some storage endpoint. 
+related to the quota usage. You may have reached your quota limit at some storage endpoint.
 You should move (or remove) the data to a different storage endpoint (or request more quota) to be able to write to the disk and then resubmit the jobs.
 \ifgent
 Another option is to request extra quota for your VO to the VO moderator/s.
-See section~\ref{subsec:predefined-user-directories} and section~\ref{subsec:predfined-quotas} for more information about 
+See section~\ref{subsec:predefined-user-directories} and section~\ref{subsec:predfined-quotas} for more information about
 quotas and how to use the storage endpoints in an efficient way.
 \fi
 % FIXME: Add how to request quota section
 
+\section{Issues connecting to login node}
+\label{sec:connecting-issues}
 
+If you have issues connecting to the login nodes, here is a list of things to do
+that might help:
 
+\begin{enumerate}
+    \item Wait at least 1 hour after you VSC account has been \emph{approved}.
+\ifmacORlinux
+    \item Your ssh private key is not in the default location (\verb|~/.ssh/id_rsa|).
```

Comment:
`is not` -> `may not be`, and please use `$HOME` rather than `~`, since it's a bit more explicit (some users have no idea what that squiggly character is supposed to mean)